express-pg-auth0/src/api/todos/repositories/__mocks__/todos.repository.ts

```diff
@@ -1,17 +1,13 @@
 import { Paginated, extractPagination } from '@utils/query';
 import { InsertTodo, ListTodosQuery, Todo, UpdateTodo } from '../../entities';
+import { createId } from '@paralleldrive/cuid2';
 
 export class TodosRepository {
-  private lastId = 0;
-  private records: Todo[] = [];
+  private records: Partial<Todo>[] = [];
```

Comment:
I guess this change comes from the previous one. In the SQL table we shouldn't have partial records